ui/components/app/custom-spending-cap/custom-spending-cap.js

```diff
@@ -231,16 +238,26 @@ export default function CustomSpendingCap({
                 : 'custom-spending-cap'
             }
           >
-            <FormField
-              inputRef={inputRef}
-              dataTestId="custom-spending-cap-input"
-              wrappingLabelProps={{ as: 'div' }}
-              id={
-                decConversionGreaterThan(customSpendingCap, currentTokenBalance)
-                  ? 'custom-spending-cap-input-value'
-                  : 'custom-spending-cap'
-              }
-              TooltipCustomComponent={
+            <Box
+              display={Display.Flex}
+              justifyContent={JustifyContent.spaceBetween}
+              alignItems={AlignItems.flexEnd}
+              marginBottom={2}
+              marginTop={1}
+            >
+              <Box
+                display={Display.Flex}
+                alignItems={AlignItems.center}
+                gap={2}
+              >
+                <Label
+                  htmlFor="custom-spending-cap"
+                  fontWeight={FontWeight.Bold}
```

Comment:
I think we can remove the fontWeight here
```suggestion
```

Reply:
> I think we can remove the fontWeight here

Removing fontWeight may cause a slight difference from the original font.

I have attached both the `before` and `after` screenshots for your review. If everything appears satisfactory, we can proceed with the changes.

<table>
<tr>
<td>original</td>
<td>before remove</td>
<td>After remove</td>
</tr>
<tr>

<td><img src="https://github.com/MetaMask/metamask-extension/assets/81921291/64431bc7-8723-4289-8101-4a2c2677c07e" /> </td>

<td> <img src="https://github.com/MetaMask/metamask-extension/assets/81921291/f1700574-855c-4cec-89f0-14cc457bf68a"/> </td>

<td> <img src="https://github.com/MetaMask/metamask-extension/assets/81921291/b75aabd3-1932-47fd-96eb-4833a033206d" /> </td>

</tr>
</table>

Reply:
Great spotting @Jainex17 the design team has update the `Label` to use medium font weight so we can align them here